dipy/direction/closest_peak.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
+from dipy.direction.peaks import peak_directions, default_sphere
+from dipy.reconst.shm import order_from_ncoef, sph_harm_lookup
+from dipy.tracking.local.direction_getter import DirectionGetter
+from dipy.tracking.local.interpolation import trilinear_interpolate4d
+
+
+def _closest_peak(peak_directions, prev_step, cos_similarity):
```

Comment:
I think, you should rename the variable `peak_directions` because it has the same name as the import above. I wonder how python manage that.

Reply:
renamed to peak_dirs

Reply:
also renamed _peak_directions to _get_peak_directions